lib/textDiff.js

```diff
@@ -2,20 +2,18 @@ exports.onStringInsert = onStringInsert;
 exports.onStringRemove = onStringRemove;
 exports.onTextInput = onTextInput;
 
-function onStringInsert(el, previous, index, text) {
+function onStringInsert(el, previous = '', index, text) {
```

Comment:
Let's not use a default param here, to keep ES5 compatibility for now, and to make for easier future TS conversion.
```js
if (!previous) {
  previous = '';
}
```